CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# cmake 3.10 required for ctest/gtest integration
+cmake_minimum_required(VERSION 3.10)
+project(usgsephem VERSION 0.0.1 DESCRIPTION "usgs ephemerides library")
```

Comment:
For further discussion later, on naming: If someone said to me "here is an ephemeris software library" I would expect to be able to use it to look up the position of an astronomical body, given a time, not to do what our software library is going to do.